.gitignore

```diff
@@ -121,6 +121,7 @@ dist
 
 # Stores VSCode versions used for testing VSCode extensions
 .vscode-test
+.vscode
```

Comment:
This vscode specific folder holds things like custom VSCode Keyboard Shortcuts. I have:
- `CTRL + SHIFT + S` bound to `npm run style` and 
- `CTRL + ALT + SHIFT + S` bound to `npm run style && git add .` to automatically style and choose all files for my next commit message